test/AutoDiff/stdlib/tgmath_derivatives.swift.gyb

```diff
@@ -114,6 +137,43 @@ DerivativeTests.test("${op}_${T}") {
 %end
     }
   }
+
+  // pow
+  let eps:${T} = 0.01
+  let ulps:${T} = eps/eps.ulp
```

Comment:
here I compute the needed `ulps` for comparing values as the resulting numerical derivative depends on the ε used to compute it.